packages/search-adapter/src/empathy/__tests__/request-mappers/empathy-queryable-request-mapper.spec.ts

```diff
@@ -1,19 +1,13 @@
-import { RelatedTag } from '@empathyco/x-types';
 import { Container } from 'inversify';
```

Comment:
As adapter files have been auto-formatted by prettier they are going to be a little harder to review, but the intention was to only remove the related tags  mapping from the request mappers.